### DISCUSSION TITLE
Adding Dominion to OpenSpiel

### DISCUSSION THREAD

**Discussion by ChrisBenka** [deepmind/open_spiel#763] [Ideas]
I am part of a team of researchers interested in adding [Dominion](http://wiki.dominionstrategy.com/index.php/Main_Page), a popular deck-building strategy game supporting 2-4 players, to the OpenSpiel framework. I wanted to confirm that the OpenSpiel team would welcome such a change and is not adding the game themselves.

Dominion's properties make the game a solid addition to the OpenSpiel framework. First, during game setup, players randomly choose ten different "kingdom cards" from over 350 other available cards. As a result, there is a lower bound of 6 * 10^18 different possible game setups. In addition, the dominion state space is small. Each game is played with only 17-25 types of cards, and these cards can only be played in a few different positions: the supply, a player's deck, their hand, or their discard. 

Please let me know if you have any questions or concerns.

**Comment by lanctot** [comment]
Hi @ChrisBenka,

I would absolutely love to see Dominion in OpenSpiel. 

For merging proprietary games, we first have to consult with our legal team. More information on the process here: https://github.com/deepmind/open_spiel/issues/770.

Please don't hesitate to reach out by personal email if you have more questions.
